go/cli/commands/build.go

```diff
@@ -114,14 +155,74 @@ func (r *BuildRunner) dockerfileExist() bool {
 	return true
 }
 
-func (r *BuildRunner) createDockerfile() error {
-	dockerfileContent, err := f.ReadFile(builtinDockerfilePath)
+func (r *DockerBuilder) createDockerfile() error {
+	dockerfileContent, err := f.ReadFile(BuiltinDockerfilePath)
+	if err != nil {
+		return err
+	}
+	if err = os.WriteFile(DockerfilePath, dockerfileContent, 0644); err != nil {
+		return err
+	}
+	fmt.Println("created Dockerfile")
+	return nil
+}
+
+func (r *KoBuilder) GuaranteeKoInstalled() error {
+	_, err := execLookPathFn("ko")
+	if err == nil {
+		return nil
+	}
+	if _, err = execCmdFn(nil, "go", "install", "github.com/google/ko@latest"); err != nil {
```

Comment:
Tip: you might want to set `GOBIN` so you can control where it goes (in case ~/go/bin is not on the path)

Reply:
Great point, Thanks!

Reply:
done